### ISSUE TITLE
[Feature request] Portable mode

### ISSUE THREAD

**Issue by ftk** [FelixdelasPozas/TrayWeather#129]
Writing settings to a file instead of windows registry would make copying the program to another computer much easier.

For example:
1. At startup check if file `portable.txt` exists in app directory and then make Qt read/write settings to `settings.ini` in app directory.
2. Publish zip/7zip archive (with no installer) which includes an empty `portable.txt` file.

**Comment by FelixdelasPozas** [comment]
Hi @ftk .

That's rather easy (at first). The application could check if a file named "TrayWeather.ini" exists in the same directory and if its writable, and the use it if so. 
By default applications in the "Program Files" folder can't write files there (the windows registry is used to store configuration) but the user can install it in another non-protected folder or the user could just copy the application files and create an empty TrayWeather.ini file in the same directory as the executable to start using it as "portable". I'll add it for the next release. 

Regards, 
Félix.

**Comment by JohnLGalt** [comment]
> Hi @ftk .
> 
> That's rather easy (at first). The application could check if a file named "TrayWeather.ini" exists in the same directory and if its writable, and the use it if so. By default applications in the "Program Files" folder can't write files there (the windows registry is used to store configuration) but the user can install it in another non-protected folder or the user could just copy the application files and create an empty TrayWeather.ini file in the same directory as the executable to start using it as "portable". I'll add it for the next release.
> 
> Regards, Félix.

If you just provide the necessary files (run) in a .ZIP that could be extracted (assuming this does not include any previously needed dependencies, like runtimes, which the user can get on their own), and pre-set it to use the .ini file, that would be superlative.I use numerous programs in 'portable' mode off an SSD on my system, so even after a clean Windows install, I have no need to set up and reconfigure the app - literally find it on the SSD, load it and all of its settings are there.

**Comment by FelixdelasPozas** [comment]
Hi @ftk .

I've just released version [1.25](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.25.0) with the portable mode. The readme has been updated with the instructions on how to "move" the registry configuration to the ini file. 
Steps:
- Copy the installation folder (by default installed as a subfolder of "Program Files", that is protected and the application cannot write to it) to another folder where the application can write. 
- Execute TrayWeather.exe 
- Create an empty "TrayWeather.ini" file in the same folder as the executable. 
- Close TrayWeather.
- The settings of the application should be now in the TrayWeather.ini file. 

From then on the application will use the ini file if present, but will look into the windows registry if the file is not present as a fallback. Tell me if it works for you and I'll close the issue then.

Regards, 
Félix.
